io.itch.tx00100xt.SeriousSamClassic-VK.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: io.itch.tx00100xt.SeriousSamClassic-VK
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: serioussam
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+
+cleanup:
+  - '*.la'
+  - '*.a'
+
+modules:
+
+  - name: serioussam
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: https://github.com/tx00100xt/SeriousSamClassic-VK/archive/refs/tags/1.10.6e.tar.gz
+        sha256: aec1d1ef3a16bb914d378d8b777d8178341253994d0e8080a514a67814039bce
+        x-checker-data:
+          type: anitya
+          project-id: 9154
+          url-template: https://github.com/tx00100xt/SeriousSamClassic-VK/archive/refs/tags/$version.tar.gz
+      - type: file
+        url: https://raw.githubusercontent.com/tx00100xt/SeriousSamClassic-VK/main/flatpak/0001-CMakeLists.txt.patch
+        sha256: df95a56b582f79bb45ef707f443bf4d2d0f8dabfa25372c309d7c2d2fc5ab2f2
+        dest-filename: 0001-CMakeLists.txt.patch
+      - type: file
+        url: https://raw.githubusercontent.com/tx00100xt/SeriousSamClassic-VK/main/flatpak/0002-Engine.cpp.patch
+        sha256: 9a7ef3408ae18cb2495a25b703533862847f81435ac6a1f5f59cb5c575c49c6e
+        dest-filename: 0002-Engine.cpp.patch
+      - type: file
+        url: https://raw.githubusercontent.com/tx00100xt/SeriousSamClassic-VK/main/flatpak/0003-desktop-file.patch
+        sha256: f0502cb566b90e8fd7a5c30939fcdf73a6deef85bf387e7238b994d23a6dfacc
+        dest-filename: 0003-desktop-file.patch
+      - type: shell
+        commands:
+          - patch -p1 < 0001-CMakeLists.txt.patch
+          - patch -p1 < 0002-Engine.cpp.patch
+          - patch -p1 < 0003-desktop-file.patch
```

Comment:
this is not how patches are handled in flatpak.

Reply:
I made local patches first. As stated here:
https://docs.flatpak.org/en/latest/manifests.html#
>- type: patch
>  path:  0001-CMakeLists.txt.patch
>- type: patch
>  path:  0002-Engine.cpp.patch
>- type: patch
>  path:  0003-desktop-file.patch
>

Here I didn’t find how to use remote patches
https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html

I think we can add patches to pull requests.
Here's how it's done:
https://github.com/flathub/org.gnome.Dictionary/blob/master/0001-fix-image-urls.patch

Reply:
Add them as `type: patch`